www/js/lib/uiUtil.js

```diff
@@ -862,15 +862,12 @@ function applyAppTheme (theme) {
 
 // Displays the return link and handles click event. Called by applyAppTheme()
 function showReturnLink () {
-    var viewArticle = document.getElementById('viewArticle');
-    viewArticle.style.display = 'block';
-    viewArticle.addEventListener('click', function (e) {
+    document.getElementById('viewArticle').style.display = 'block';
+    document.getElementById('viewArticle').addEventListener('click', function (e) {
         e.preventDefault();
         document.getElementById('liConfigureNav').classList.remove('active');
         document.getElementById('liHomeNav').classList.add('active');
-        tabTransitionToSection('home', params.showUIAnimations);
-        const welcomeText = document.getElementById('welcomeText');
-        welcomeText.style.display = 'none';
+        viewArticle();
         viewArticle.style.display = 'none';
```

Comment:
In your current code, this line is causing an error (see below). Because viewArticle is a function, not a DOM element.

![image](https://github.com/kiwix/kiwix-js/assets/4304337/2b1eb841-856e-4608-82ea-ce38c174eeee)